### ISSUE TITLE
STICK_AXIS_Y refuses to invert

### ISSUE THREAD

**Issue by GormAuslander** [Electronicks/JoyShockMapper#25]
Fairly simply, I tried adding the argument `STICK_AXIS_Y = INVERTED` to the end of the included _2Dmouse.txt config, and it doesn't invert. Am I doing it wrong or is it broken?

```RIGHT_STICK_MODE = AIM
STICK_SENS = 360
STICK_POWER = 1
STICK_ACCELERATION_RATE = 2
STICK_ACCELERATION_CAP = 4
STICK_AXIS_Y = INVERTED`

**Comment by GormAuslander** [comment]
Thanks!

**Comment by Electronicks** [comment]
Fixed in v3.2.2
